.github/workflows/release.yml

```
@@ -0,0 +1,40 @@
+name: Release to npm
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'The version number to tag and release'
+        required: true
+        type: string
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    environment: release
+    permissions:
+      contents: write
+      id-token: write
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '20'
+          registry-url: 'https://registry.npmjs.org'
+      - run: npm install -g npm@latest
+      - run: npm install
+      - name: Change version number and push
+        run: |
+          git config --global user.email "hello@matteocollina.com"
+          git config --global user.name "Matteo Collina"
+          npm version ${{ inputs.version }} --git-tag-version=false
+          git push origin HEAD:main
```

should we push the new version into GH after `npm publish`?

```suggestion
          git add -u
          git push origin HEAD:main
```

---

`npm version` should add a commit already.

---

I literally just learned about `npm version` 🙈

I was doing manual commits before 🙈

---

I second @gurgunday, literally TIL about `npm version` doing that 😅